src/style.css

```diff
@@ -1,3 +1,76 @@
+/* #region Reset */
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+a {
+  cursor: pointer;
+}
+/* #endregion */
+
+.navigation {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: auto;
+  padding: 0 50px;
+}
+
+.nav-logo {
+  display: block;
+  margin-top: 5px;
+}
+
+.nav-list {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  text-transform: uppercase;
+  white-space: nowrap;
+  margin: 0;
+  padding: 0;
+}
+
+.link {
+  color: black;
+  text-decoration: none;
+  display: block;
+  position: relative;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
```

Comment:
it's bad practice to set paddings like 22.5px, better set height for link - 60px,  it would work properly if you  change display to flex and set align-items to center

Reply:
Thank you, but in task says, that we shouldn't fix height, and also everything is on wrong position, I tryed,  but I prefer not to fight against this test, I prefer to move on next tasks, and there I will try remembering this rule. :)